src/test/java/com/aws/greengrass/util/platforms/unix/UnixPlatformTest.java

```diff
@@ -5,28 +5,100 @@
 
 package com.aws.greengrass.util.platforms.unix;
 
+import com.aws.greengrass.logging.api.Logger;
+import com.aws.greengrass.logging.impl.LogManager;
 import com.aws.greengrass.testcommons.testutilities.GGExtension;
 import com.aws.greengrass.util.FileSystemPermission;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.condition.DisabledOnOs;
 import org.junit.jupiter.api.condition.OS;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
 
+import java.io.IOException;
 import java.nio.file.attribute.PosixFilePermission;
+import java.util.Arrays;
 import java.util.Set;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.arrayContaining;
 import static org.hamcrest.Matchers.contains;
 import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.empty;
 import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
-@ExtendWith({GGExtension.class})
+
+@ExtendWith({MockitoExtension.class, GGExtension.class})
 @DisabledOnOs(OS.WINDOWS)
 class UnixPlatformTest   {
 
     private static String[] command = {"echo", "hello", "world"};
+    private static final Logger logger = LogManager.getLogger(UnixPlatformTest.class);
+
+    @Spy
+    private final UnixPlatform spyPlatform = new UnixPlatform();
+    private final static String user = "ggc_user01";
+    private final static String group = "ggc_group01";
+
+    @Test
+    public void GIVEN_busybox_environment_WHEN_add_user_THEN_add_user_success() {
+        try {
+            spyPlatform.runCmd("useradd -r -m " + user, o -> {
+            }, "Failed to create user with useradd");
+        } catch (IOException e) {
+            logger.info(e.getMessage());
+        }
+
+        try {
+            spyPlatform.createUser(user);
```

Comment:
Same for this, seems like everything you're testing is in the catch block below?